docs: explain windows 10 alg limitation

cpb8010 opened the pull request:
# Description

Because windows 10 only supports RS256, it's not supported by our ES256
verifier, so until we add an RS256 verifier this platform won't work.
We've attempted to limit these sign-ups via the browser, but it doesn't
appear to stop users from choosing that option. Documenting it is the
next best thing, but we'll probably add a more explicit error next.

## Linked Issues

https://github.com/matter-labs/zksync-account-sdk/issues/182

## Additional context

The current NFT demo (as of this writing) will also fail for Windows 11, so it requires a small update so these docs are correct

keriat commented:
:tada: This PR is included in version 2.19.0 :tada:

The release is available on [GitHub release](https://github.com/matter-labs/zksync-docs/releases/tag/2.19.0)

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:

cpb8010 committed:
docs: explain windows 10 alg limitation

Because windows 10 only supports RS256, it's not supported by our ES256
verifier, so until we add an RS256 verifier this platform won't work.
We've attempted to limit these sign-ups via the browser, but it doesn't
appear to stop users from choosing that option. Documenting it is the
next best thing, but we'll probably add a more explicit error next.
